src/main/resources/api/service_openapi.yaml

```diff
@@ -60,6 +60,19 @@ paths:
           $ref: '#/components/responses/PermissionDenied'
         '500':
           $ref: '#/components/responses/NotFound'
+    get:
```

Comment:
I think this should have offset and limit for paging, and use a sorted query in the DAO to keep the list close to consistent. There is an example in the file already in the deprecated `enumerateReferences` endpoint.

Reply:
done. I originally skipped this because the underlying Sam call doesn't support any pagination/filtering, but it's probably useful for WSM callers